packages/eslint-config/src/configs/vue.ts

```
@@ -0,0 +1,137 @@
+						: null,
+					sourceType: 'module',
+				},
+			},
+			name: 'mheob/vue/rules',
+			processor: pluginVue.processors['.vue'],
+			rules: {
+				...pluginVue.configs.base.rules,
+
+				...pluginVue.configs['flat/essential']
+					.map(c => c.rules)
+					.reduce((accumulator, c) => ({ ...accumulator, ...c }), {}),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)